horizon_net/horizonnet_reconstruction.py

```diff
@@ -73,16 +79,41 @@ def augment_undo(x_imgs_augmented, aug_type):
     return np.array(x_imgs)
 
 
-class horizonNet:
-    def __init__(self, model_path=None):
-        if model_path is None:
-            model_path = STAGED_MODEL_DIRNAME / MODEL_FILE
-        self.model = torch.jit.load(model_path)
+class HorizonNet:
+    """Trained HorizonNet Model Class
+
+    Loads the pretrained torchscript deployed memory from a public s3 bucket and
+    executes a prediction over new unseen data.
+    """
+
+    def __init__(self):
+        if os.path.isfile(MODEL_FILE) is False:
+            print("downloading", os.getcwd())
+            _ = request.urlretrieve(MODEL_URL, MODEL_FILE)
+            print("downloaded", os.getcwd(), os.listdir())
+        self.model = torch.jit.load(MODEL_FILE)
 
     @torch.no_grad()
-    def predict(self, path):
+    def predict(self, image: Union[str, Path, Image.Image]):
```

Comment:
Possibly a noob question but is there a return statement missing for this method?

Reply:
Thought it might be a noob question!